src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -518,6 +516,34 @@ public void setProcessOOB(boolean processOOB) {
     getSession().processOOB = processOOB;
   }
 
+  /**

```

Comment:
These following changes seem unrelated to the localization support.

Reply:
one very small addition - I don't want to split up ProgramAB into parts for this commit - originally I was working on ProgramAB, before I got squirreled with localize